dask-xfails.txt

```diff
@@ -0,0 +1,139 @@
+# This fails in dask
+# import dask.array as da
+# a = da.array([1]).reshape((1,1))
+# key = (0, slice(None, None, -1))
+# a[key] = da.array([1])
+
+# Failing hypothesis test case
+#x=dask.array<zeros_like, shape=(0, 2), dtype=bool, chunksize=(0, 2), chunktype=numpy.ndarray>
+#| Draw 1 (key): (slice(None, None, None), slice(None, None, None))
+#| Draw 2 (value): dask.array<zeros_like, shape=(0, 2), dtype=bool, chunksize=(0, 2), chunktype=numpy.ndarray>
+
+# TODO: this also skips test_setitem_masking unnecessarily
```

Comment:
Let's change the xfails file matching to require an exact match. No reason not to do that.

Reply:
Should I make a PR into array-api-tests?

Reply:
Yes go ahead. It should be a trivial change. Just change some `in`s to `==` in conftest.py, and make sure some existing xfail files don't need to be updated.

Reply:
So, it turns out that after fixing some other tests, this test(``test_setitem_masking``) started failing again (the test was passing because of a bug I had in my asarray version), and the comment isn't relevant anymore.